src/openrct2-ui/input/KeyboardShortcut.cpp

```diff
@@ -592,11 +592,11 @@ static void shortcut_quick_save_game()
     }
     else if (gScreenFlags & SCREEN_FLAGS_SCENARIO_EDITOR)
     {
-        Intent * intent = intent_create(WC_LOADSAVE);
-        intent_set_uint(intent, INTENT_EXTRA_LOADSAVE_TYPE, LOADSAVETYPE_SAVE | LOADSAVETYPE_LANDSCAPE);
-        intent_set_string(intent, INTENT_EXTRA_PATH, gS6Info.name);
+        Intent * intent = new Intent(WC_LOADSAVE);
```

Comment:
You don't need to heap-allocate Intents using `new` if you're calling their destructor (`delete`) within the same function. Just keep them on the stack.

Reply:
This is how it was done before. I only removed the wrappers.

Reply:
Also use `auto` for the type instead.